source/IrcDotNet/IrcClientMessageProcessing.cs

```diff
@@ -27,8 +27,12 @@ protected internal void ProcessMessageNick(IrcMessage message)
 
             // Local or remote user has changed nick name.
             Debug.Assert(message.Parameters[0] != null);
-            sourceUser.NickName = message.Parameters[0];
-        }
+
+			string oldNickName = sourceUser.NickName;
```

Comment:
Use `var` wherever possible. Also, indentation messed up here, I think?

Reply:
Maybe it's a spaces vs. tabs issue, and that needs to be normalised? Check for that, and let's maybe add a `.editorconfig` file too.

Reply:
Ok, it's probably due to VisualStudioCode issue.
I'm using var on a non primitive types(with the exception of string) to be more explicit. If your Coding Standard are var everywhere i can fix that to be align. Although i'm big supporter of writing as Microsoft writes.

Reply:
Yeah, I prefer `var`everywhere, and AFAIK, that's the convention I used up to this point. Any half-decent IDE or editor like VS/VScode shows you the type anyway. :-)